web/js/components/timeline/timeline-draggers/timeline-dragger.js

```diff
@@ -38,29 +39,49 @@ class Dragger extends PureComponent {
   * @returns {void}
   */
   handleDragDragger = (e, d) => {
+    const { axisWidth, draggerPosition, handleDragDragger } = this.props;
+    const { deltaX, x } = d;
+    const draggerAtStart = x <= -15;
+    const draggerAtEnd = x + 36 >= axisWidth;
+
+
     this.setState({
       isHoveredDragging: true,
     });
-    this.props.handleDragDragger(e, d);
+
+    // if dragger is positioned near axis edge from panning axis vs. dragging dragger
+    if (draggerPosition <= -15 || draggerPosition + 49 + 36 >= axisWidth) {
```

Comment:
Could add `-15`, `36`, and `49` as variables (so someone who touches with this in the future will know what these numbers represent )
e.g. `const 36 = timelinePaddingRight `